src/app/index.scss

```diff
@@ -9,17 +9,121 @@ $fa-font-path: "../../bower_components/fontawesome/fonts";
 @import "../../bower_components/fontawesome/scss/font-awesome.scss";
 @import "../../bower_components/angular-tooltips/dist/angular-tooltips";
 
+
 .top{
   z-index: 100;
 }
 
+/* prevent element from being selected http://stackoverflow.com/questions/826782 */
+.noselect {
+  user-select: none;
+  cursor: default;
+}
+
+.voyager {
+  background-color: #ECF1F7;
+}
+
+.voyager a {
+  color: #38425D;
+}
+
+.voyager a:hover {
+  color: #72757B;
+}
+
+h1 {
```

Comment:
as i modified h2 for voyager - i thought it would make sense to just keep all the headers together. I can move them back to vlui though.